signinapp/events.py

```diff
@@ -94,7 +94,10 @@ def validate(self):
 @events.route("/events")
 @mentor_required
 def list_events():
-    events: Event = db.session.execute(select(Event).filter_by(enabled=True)).scalars()
+    events: list[Event] = db.session.execute(
```

Comment:
Can we not let sql handle the sorting? The adjusted time won't change the order at all.